src/stan/io/util.hpp

```diff
@@ -16,14 +16,12 @@ namespace io {
  * @return String representation of current UTC.
  */
 std::string utc_time_string() {
-  // FIXME:  use std::strftime
-
   // original with asctime
```

Comment:
Notice below where strftime is called. This fixme has already been fixed and can be deleted.